flink-python/pyflink/table/tests/test_pandas_udaf.py

```diff
@@ -107,10 +107,10 @@ def test_group_aggregate_with_aux_group(self):
                                                      result_type=DataTypes.INT(),
                                                      func_type="pandas"))
         self.t_env.create_temporary_system_function("mean_udaf", mean_udaf)
-        t.group_by("a") \
-            .select("a, a + 1 as b, a + 2 as c") \
-            .group_by("a, b") \
-            .select("a, b, mean_udaf(b), max_add(b, c, 1)") \
+        t.group_by(t.a) \
+            .select(t.a,  (t.a + 1).alias("b"), (t.a + 2).alias("c")) \
+            .group_by(t.a, t.b) \
+            .select(t.a, t.b, mean_udaf(t.b), expr.call("max_add", t.b, t.c, 1)) \
```

Comment:
can't we import `expr` as we did in `flink-end-to-end-tests/flink-python-test/python/python_job.py` to call `call` and `col` directly?